core/src/components/breadcrumbs/test/standalone/breadcrumbs.e2e.ts

```diff
@@ -0,0 +1,13 @@
+import { expect } from '@playwright/test';
+import { test, Viewports } from '@utils/test/playwright';
+
+test.describe('breadcrumbs: standalone', () => {
+  test('should not have visual regressions', async ({ page }) => {
+    await page.goto(`/src/components/breadcrumbs/test/standalone`);
+
+    // avoid contents getting cut off
+    await page.setViewportSize(Viewports.tablet.portrait);
```

Comment:
Are you able to do `page.screenshot({ fullscreen: true })` instead? Since there is no `ion-content` in the standalone file, that should work fine.